### ISSUE TITLE
[question] Python 3.8 support 

### ISSUE THREAD

**Issue by superpoussin22** [frida/frida#1057]
Hello

did you plan to create Python package for Python 3.8 ?

**Comment by oleavr** [comment]
There is no need, as the 3.x C ABI is stable – so you can use our precompiled 3.6/3.7 binaries on 3.8 also.

**Comment by wdscxsj** [comment]
A 3.8 binary for Windows is still necessary, I suppose?

**Comment by oleavr** [comment]
Yes unfortunately there is a hard-coded reference containing the version number. For now you can change that using a hex-editor as a workaround.

**Comment by wdscxsj** [comment]
Thanks for the tip!

**Comment by mincadev** [comment]
I hope to have support in python 3.8 soon

**Comment by wdscxsj** [comment]
@oleavr Python 3.8 support is really necessary. If only one version could be distributed in binary form but it's not the latest stable version, it looks weird...

**Comment by oleavr** [comment]
@wdscxsj No there is no such constraint, but it would be a waste to upload multiple copies with only one byte difference. Our `setup.py` fakes the build process by downloading the .egg, so we can support both legacy and modern installation methods. We recently added a [quirk](https://github.com/frida/frida-python/blob/9671514a8bd7cf69e2970e054eafc1f26185cf76/setup.py#L139-L140) that does the fixup on Windows – we should be able to implement the same kind of hack for Linux as well. Shouldn't be needed on macOS though. Will take a quick stab at this later today, and push a new release if it's as easy as it looks.

**Comment by wdscxsj** [comment]
@oleavr Thanks, and I'm aware of the hack mentioned in your reply before. But let's suppose frida will be used and loved by millions of people.

- Do the fix by end users:
  - Cons: tens of seconds wasted per user (manual download + manual fix); beginner unfriendly; hard to follow the quick releases.
  - Pro: nothing has to change.
- Do the fix on the server side:
  - Cons: cost of a few megabytes of public resource; one-time change to the release workflow.
  - Pros: familiar pip install/upgrade user experience; easy to keep up with the new versions; more Pythonic?

**Comment by oleavr** [comment]
@wdscxsj Oh I never meant to imply that end users are supposed to patch up their binary locally. That was only suggested as a temporary workaround until we've fixed this issue. I'm spread too thin so just haven't gotten to this yet – but PRs are always welcome.
